webdriver-spec.html

```diff
@@ -1888,16 +1888,17 @@
 </table>
 
 <p>The <dfn>Get Current Url</dfn> <a>command</a>
- returns the URL of the <a>current top-level browsing context</a>.
+ returns the URL of the <a>current browsing context</a>.
```

Comment:
This will return the src of a frame which is not what is intended by this call. This is supposed to return what is in the URL bar